keyboards/deemen17/de60fs/info.json

```diff
@@ -0,0 +1,126 @@
+{
+    "manufacturer": "Deemen17",
+    "keyboard_name": "De60fs",
+    "maintainer": "Deemen17",
+    "bootloader": "rp2040",
+    "build": {
+        "debounce_type": "sym_defer_pk"
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "indicators": {
+        "caps_lock": "GP24"
+    },
+    "matrix_pins": {
+        "cols": ["GP23", "GP22", "GP21", "GP20", "GP28", "GP0", "GP8", "GP7", "GP6", "GP5", "GP4", "GP3", "GP2", "GP1"],    
+        "rows": ["GP27", "GP26", "GP25", "GP29", "GP18"]
+    },
+    "processor": "RP2040",
+    "rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        },
+        "led_count": 22,
+        "max_brightness": 240,
+        "sleep": true
+    },
+    "usb": {
+        "vid": "0xDE17",
+        "pid": "0x60F0",
+        "device_version": "0.0.1"
+    },
+    "ws2812": {
+        "driver": "vendor",
+        "pin": "GP11"
+    },
+    "community_layouts": ["60_tsangan_hhkb"],
+    "layouts": {
```

Comment:
would it be correct that matrix co-ordinates for following keys are?:
2u Backspace: [0, 13]
2.75u Right Shift: [3, 12]

Reply:
Yes, it is.

Reply:
Thank you, please see community layout [suggestion](https://github.com/qmk/qmk_firmware/pull/22667#discussion_r1428739398)